HarmonyCore/FileIO/Queryable/QueryBuffer.dbl

```diff
@@ -234,7 +234,16 @@ namespace Harmony.Core.FileIO.Queryable
 				allBuffers = allBuffers.Concat(DrivingBuffers)
 
 			if(FailedDueToNetwork)
-				DataObjectProvider.ChannelManager.AbandonOpenChannels()
+			begin
+				;abandon channels for a particular connection only
```

Comment:
Im not sure this does what the comment says it does. It sort of looks like its just closing all the channels in use for this particular query buffer. We want to abandon all channels that used this same failed network socket.

Reply:
Did you review the changes in AbandonOpenChannels() in FileChannelManager.dbl? I'm passing a channel number to AbandonOpenChannels, and in AbandonOpenChannels I'm using channel number to identify a connection. Then, I'm removing all of the channels associated with that connection.

Reply:
Ah, didnt see that but it makes sense now